DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py

```diff
@@ -1,4 +1,63 @@
 import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+
+
+#george
+double_soft_muon_muonpt_efficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/SUSY/SOS/Muon/*"),
```

Comment:
I see, you have many instances of the same class
in order to be on the plateau of the other leg(s)
right ?